documentation/book/proc-configuring-kafka-mirror-maker-authentication-scram-sha-512.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='proc-configuring-kafka-mirror-maker-authentication-scram-sha-512-{context}']
+= Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator with a `listener` configured for SCRAM-SHA-512 authentication
+* Username to be used for authentication
+
+.Procedure
+
+Because Kafka Mirror Maker connects to two Kafka clusters (source and target), it is possible to configure SCRAM-SHA-512 authentication for one or both of them.
+Following the steps for configuring SCRAM-SHA-512 authentication on the consumer side for connecting to the source Kafka cluster.
+
+. Find out the name of the `Secret` with the password which should be used for authentication and the key under which the password is stored in the `Secret`.
+If such a `Secret` does not exist yet, prepare a file with the password and create the `Secret`.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl create`:
+[source,shell,subs=+quotes]
+echo -n '_<password>_' > _<my-password.txt>_
+kubectl create secret generic _<my-secret>_ --from-file=_<my-password.txt>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc create`:
++
+[source,shell,subs=+quotes]
+echo -n '1f2d1e2e67df' > _<my-password.txt>_
+oc create secret generic _<my-secret>_ --from-file=_<my-password.txt>_
+. Edit the `KafkaMirrorMaker.spec.consumer.authentication` property.
+For example:
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker
+metadata:
+  name: my-mirror-maker
+spec:
+  # ...
+  consumer:
+    authentication:
+      type: scram-sha-512
+      username: _<my-username>_
+      passwordSecret:
+        secretName: _<my-secret>_
+        password: _<my-password.txt>_
+  # ...
+----
++
+. Create or update the resource.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl apply -f _<your-file>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc apply -f _<your-file>_
+
+The same steps can be done for configuring SCRAM-SHA-512 authentication on the producer side for connecting to the target Kafka cluster.
```

Comment:
Rewrite to: 

Repeat the above steps for configuring SCRAM-SHA-512 authentication on the target Kafka cluster. In this case, the secret containing the certificate has to be configured in the KafkaMirrorMaker.spec.producer.tls property.